src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,3 +1,11 @@
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+import React from 'react';
+import { User } from '../../type';
+
+import './UserInfo.scss';
+
+export const UserInfo: React.FC<{ user: User }> = ({ user }) => (
```

Comment:
Let's follow this approach, it's easily to read
interface Props {
  ...
}
```suggestion
export const UserInfo: React.FC<Props> = ({ user }) => (
```